orga/app/components/auth/login-form.js

```diff
@@ -59,8 +57,26 @@ export default class LoginForm extends Component {
   }
 
   @action
-  togglePasswordVisibility() {
-    this.isPasswordVisible = !this.isPasswordVisible;
+  validatePassword(event) {
+    this.password = event.target.value;
+    const isInvalidInput = isEmpty(this.password);
```

Comment:
Le message d'erreur `this.intl.t('pages.login-form.errors.empty-password')` est spécifique, il n'y a qu'une condition à vérifier l'absence de pwd, je suggère de renommer cette constante en `isEmptyInput`

Reply:
c'est un chipotage ou un vrai problème de lecture ? Comme isEmpty est le nom de la fonction lodash, je vois pas trop d’intérêt à le renommer ici